### DISCUSSION TITLE
Remote access to Photoview

### DISCUSSION THREAD

**Discussion by lazycatze** [photoview/photoview#767] [Q&A and Troubleshooting] [answered]
Hi all,

I checked a lot of topics here but cannot find the correct information. Is it possible to get remote access to Photoview i.e. outside of the home network?

I installed Photoview on my NAS and it works pretty well. I also installed an iOS app and everything is ok with the access inside the home network.

But it's impossible to connect using the external IP address of my NAS.

Could someone please share a guide: how to configure remote access to the Photoview? Thanks.

**Comment by stephenp** [comment, marked as answer]
Since every router is slightly different, use the search term _your router model_ plus "opening ports" on google.  

You'll want to open the ports you've specified Photoview to use and direct them to the internal IP address of your NAS. That should get you up and running.

**Reply by lazycatze** [reply]
That’s the solution! Thank you very much
